scripts/install_third_party_libs.py

```diff
@@ -69,6 +69,25 @@
     common.OPPIA_TOOLS_DIR, 'pylint-quotes-0.1.8', 'configparser.py')
 
 
+def tweak_yarn_executable():
+    """When run yarn on Windows, the file `yarn` will be excecuted (
+    even then command specified is `yarn.cmd`). While this file is a bash
+    script, so it can not be executed. To avoid the issue, we can rename
+    this file to `yarn.sh`.
+    """
+    origin_file = os.path.join(common.YARN_PATH, 'bin', 'yarn')
+    if os.path.isfile(origin_file):
+        renamed_file = os.path.join(common.YARN_PATH, 'bin', 'yarn.sh')
+        os.rename(origin_file, renamed_file)
+
+
+def get_yarn_command():
+    """Get the executable file for yarn."""
+    if common.is_windows_os():
+        return 'yarn.cmd'
```

Comment:
Is this supposed to be yarn.sh or yarn.cmd (based on the docstring above)?

Reply:
`yarn.cmd`, `sh` file can not be executed on Windows (if you don't explicitly let it run via bash).